modules/javafx.graphics/src/main/java/javafx/scene/AccessibleAttribute.java

```diff
@@ -689,6 +691,7 @@ public enum AccessibleAttribute {
 
     /**
      * Returns the text selection start offset for the node.
+     * Selection start corresponds to the smaller offset in the selection range.
```

Comment:
Just an observation, The selection is represented by IndexRange where we refer the start and end as **index**. But here in this file, we refer them as offset.

Reply:
Hmm, actually, `IndexRange.getStart()` L76 and `.getEnd()` L84 refer to start/end _position_.

Reply:
I was just sharing an observation, wasn't expecting any change.
In this file, documentation already refer them as offset at a few places, like BOUNDS_FOR_RANGE, LINE_START, LINE_END.
So, offset seems more suitable in this file.

Reply:
ok, will revert.